cncnet-api/app/Commands/FindOpponent.php

```diff
@@ -137,8 +138,16 @@ public function handle()
         foreach ($opponentEntries as $opponentEntry)
         {
             $oppPlayer = $opponentEntry->qmPlayer->player;
+            $oppPlayerCache = PlayerCache::where("player_id", $oppPlayer->id)->first();
             $oppUserSettings = $oppPlayer->user->userSettings; //opponent's point filter flag
 
+            # Checks players are in same league tier otherwise skip
+            if ($oppPlayerCache->tier !== $playerCache->tier)
```

Comment:
does this apply to all ladders then? or just blitz?

Reply:
All ladders, but all players tiers are 1 by default so shouldn't be an issue